scripts/build

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+ZIP_NAME="deployment.zip"
```

Comment:
We should change this to a variable that can be overridden by env var if/when we generalize this pattern.